### ISSUE TITLE
Could you add a Demo of iOS?

### ISSUE THREAD

**Issue by wxlpp** [NanoMichael/cLaTeXMath#3]

**Comment by NanoMichael** [comment]
Sorry to read your issue until today cause I didn't receive any mail from Github. Actually, I'm not an iOS developer, so I can't give you a demo directly. I suggest you check out the code under the directory `src/platform/cario/` that is a sample graphics implementation under the Linux OS or `src/platform/gdi_win` that under Windows OS, and the directory `src/samples/` that are the application main entries of these platforms. I can give you some advice to write a demo of iOS.

Basically, you need to implement the `abstract graphics interface` that is defined in the file `src/graphic/graphic.h`, please check it out, it is very simple, just contains the very basic operations of a 2D graphics environment, you just need to wrap the graphics operation in `iOS` into `C++` implementation, I guess.

And next, you need to package this project into iOS formatted native library. Please let me know if you have any questions, have fun.
